### ISSUE TITLE
Some problems interpreting code ?!?

### ISSUE THREAD

**Issue by andretietz** [hexojs/hexo-renderer-markdown-it#144]
I just noticed on my websited, following markdown is not correctly interpreted into html:
https://gist.githubusercontent.com/andretietz/2d54504084afb2a9b81ddc09cd24b3cc/raw/48e27e5d5dc5f25070e4d34a431276a67a7c5126/TEST.md

Also, I noticed that every linebreak in markdown is translated into an html linebreak. Even though it shouldn't.
This is important for writing markdown, since at the moment I need to write every thing in a single line, which I don't want to have
linebreaked:

https://gist.github.com/shaunlebron/746476e6e7a4d698b373

**Comment by YoshinoriN** [comment]
> I just noticed on my website, linked markdown is not correctly interpreted into html. (See website article with the problem.)

Would you please try remove space from below line?

```diff
  override fun adapt(call: Call<Any>): RETURN_TYPE {
    registration[RequestIdentifier.identify(call.request())] = info
    return adapter.adapt(call)
  }
}
+``` 
-```
```

**Comment by andretietz** [comment]
Indeed, I've overseen this. It solved the first problem (at least on my website)

The problem with the linebreaks still remaining.

**Comment by andretietz** [comment]
Also found the problem with the line break.

by configuration linebreaks [will be directly translated into <br />](https://github.com/hexojs/hexo-renderer-markdown-it/blob/master/index.js#L14)

Compare with the option for linebreaks [here](https://markdown-it.github.io/#md3=%7B%22source%22%3A%22This%5Cnis%5Cna%5Cnlinebreak%5Cnafter%5Cnthe%5Cnother%5Cn%22%2C%22defaults%22%3A%7B%22html%22%3Atrue%2C%22xhtmlOut%22%3Afalse%2C%22breaks%22%3Afalse%2C%22langPrefix%22%3A%22language-%22%2C%22linkify%22%3Atrue%2C%22typographer%22%3Atrue%2C%22_highlight%22%3Atrue%2C%22_strict%22%3Afalse%2C%22_view%22%3A%22html%22%7D%7D)

**Comment by NosedN** [comment]
> Also found the problem with the line break.

You can change`node_modules/hexo-renderer-markdown-it/index.js`  such as
`hexo.config.markdown.render = Object.assign({
  html: true,
  xhtmlOut: false,
- breaks: true,
+breaks: false,
  linkify: true,
  typographer: true,
  quotes: '“”‘’'
}, hexo.config.markdown.render);`
